addons/sys_server/fnc_onGetRadioId.sqf

```diff
@@ -18,7 +18,7 @@
 
 params ["_player", "_class", "_callback", ["_replacementId",""]];
 
-if (getNumber (configFile >> "CfgWeapons" >> _class >> "acre_hasUnique") == 0) then {
+if (!(_class call EFUNC(sys_radio,isBaseClassRadio))) then {
```

Comment:
`if !(_class call EFUNC(sys_radio,isBaseClassRadio)) then {` makes it easier to read IMO.